src/app/ApplicationTemplate.Shared.Views/Styles/Controls/CheckBox.xaml

```diff
@@ -499,7 +498,8 @@
 										  HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
 										  VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
 										  Grid.Column="1"
-										  TextWrapping="Wrap" />
+										  TextWrapping="Wrap"
```

Comment:
Why are we using `TextWrapping="Wrap"`?

Reply:
i assume that is to avoid long text breaks the UI :/